cmd/skywire-visor/commands/root.go

```diff
@@ -581,3 +446,144 @@ func checkHvIsRunning(addr string, retries int) bool {
 	}
 	return false
 }
+
+func updateConfig(mLog *logging.MasterLogger, conf *visorconfig.V1) {
```

Comment:
I am wondering if it would be better to ask the user to run the update CLI command. For most users we will perform config upgrades via package scripts. For others running from source, they could/should be able to run CLI I would think. Would save us the trouble from reimplementing the CLI code here.

Reply:
I personally never use config update.

Config gen is very solidly implemented, whereas it isn't clear that config update will act as desired and was otherwise untested for updating possibly between versions. Its intended use was only to change the values that cannot be directly set with config gen.